packages/example-getting-started/docs/4-todo-model.md

```diff
@@ -0,0 +1,104 @@
+### Building the Todo model
+
+The Todo model will be the object we use both as a Data Transfer Object (DTO) on
+the controller, and as a LoopBack model for the Legacy Juggler implementation.
+
+Create another folder in `src` called `repositories` and inside of that folder,
+create two files:
+- `index.ts`
+- `todo.repository.ts`
+
+>**NOTE:**
+The `index.ts` file is an export helper file; this pattern is a huge time-saver
+as the number of models in your project grows, because it allows you to point
+to the _directory_ when attempting to import types from a file within the target
+folder. We will use this concept throughout the tutorial!
+```ts
+// in src/models/index.ts
+export * from './foo.model';
+export * from './bar.model';
+export * from './baz.model';
+
+// elsewhere...
+
+// with index.ts
+import {Foo, Bar, Baz} from './models';
+// ...and without index.ts
+import {Foo} from './models/foo.model';
+import {Bar} from './models/bar.model';
+import {Baz} from './models/baz.model';
+```
+
+In our Todo model, we'll create a basic representation of what would go in
+a Todo list. Our model will include:
+- a unique id
+- a title
+- a description that details what the todo is all about
+- a boolean flag for whether or not we've completed the task.
+
+For the Legacy Juggler to understand how to work with our model class, it
```

Comment:
Not a grammar master, but we might need a comma here: `Legacy Juggler, to`. Or we might need to change the phrasing so that we don't have too many commas in this sentence.